app/controllers/chefs/ingredients_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Chefs::IngredientsController < ApplicationController
+  #about 10 mins left. Feeling uneasy about how I structured things now. 
+  # Haven't had much practice with nested or namespaced routes. 
+  # why is it giving a routing error for uninitialized constant IngredientsController?
+end
```

Comment:
I actually like that you nested this Ingredients Controller inside a chefs directory. You don't have to, but it makes sense to me since your routes are nested. 
The reason you're seeing the uninitialized constant error, is because you haven't told your routes to look into this nested file for the IngredientsController, so rails is just looking in the controller dir. You'd have to go into your routes and hand roll the part `get "/chef/:chef_id/ingredients", to: "chefs/ingredients#index"` You can also see this in your terminal when you type `rails routes`, the `chefs_ingredients_path` goes to the `ingredients#index` so NOT the nested one. 
Alternatively, you could choose to not put this controller in the chefs dir and just have it in the controller dir.